src/components/TradingStatePanel/TradingStatePanel.container.js

```diff
@@ -4,15 +4,17 @@ import { reduxSelectors } from '@ufx-ui/bfx-containers'
 import {
   getFilteredPositionsCount,
   getFilteredAtomicOrdersCount,
-  getFilteredAlgoOrdersCount,
```

Comment:
seems this selector `get_filtered_algo_orders_count.js` isn't used elsewhere in code and should be removed